Makefile

```diff
@@ -126,20 +147,42 @@ sign-cosigned:
 sign-sget:
 	cosign sign --key .github/workflows/cosign.key -a GIT_HASH=$(GIT_HASH) ${KO_PREFIX}/sget:$(GIT_HASH)
 
+
+##################
```

Comment:
I am wondering if moving this piece into a release.mk and the above to a ci.mk would make our life easier here.

Reply:
sounds good to me

Reply:
made some changes to address your feedback, please take another look when you have some time, thanks again!